### DISCUSSION TITLE
OnResume is not working for Postgresql

### DISCUSSION THREAD

**Discussion by Ramyacheva** [elsa-workflows/elsa-core#4038] [Q&A]
I have tried to call onResume method for custom blocking activity but I am getting below error

Exception Details
{
 "Type": "System.NullReferenceException",
 "Message": "Object reference not set to an instance of an object.",
 "StackTrace": "   at Elsa.Server.ManulaActivity.ApprovalActivities.ApprovalActivity.OnResume(ActivityExecutionContext context)\r\n   at Elsa.Services.Activity.OnResumeAsync(ActivityExecutionContext context)\r\n   at Elsa.Services.Activity.ResumeAsync(ActivityExecutionContext context)\r\n   at Elsa.Services.Workflows.WorkflowRunner.<>c.<.cctor>b__19_1(ActivityExecutionContext context, IActivity activity)\r\n   at Elsa.Services.Workflows.WorkflowRunner.TryExecuteActivityAsync(ActivityOperation activityOperation, ActivityExecutionContext activityExecutionContext, IActivity activity, CancellationToken cancellationToken)",
 "InnerException": null,
 "Data": {}
}

Here I am giving the code snippet for onresume:
//callling this to resume the task
//await _workflowLaunchpad.ExecutePendingWorkflowAsync(WID, activityId, null, cancellationToken);

// OnResume method will get called from here as per our understanding

` protected override IActivityExecutionResult OnResume(ActivityExecutionContext context)
        {
           
            _dataLayer.NgUpdateApprovalRequest(context.WorkflowInstance.Id.ToString(),context.ActivityId.ToString(), "Actioned");
       
            return Done();
        }
`

**Comment by sfmskywalker** [comment]
It's hard to see what's going on with custom code, so if you can it would be better to provide a (simplified) project that reproduces the issue that we can run.

You mentioned in your title "Postgresql" - does this mean that it does work for e.g. SQLite?
